protocols/meson.build

```diff
@@ -20,6 +20,7 @@ protocols = [
 	['idle.xml'],
 	['wlr-input-inhibitor-unstable-v1.xml'],
 	['wlr-output-power-management-unstable-v1.xml'],
+	['wp-surface-scale-v1.xml'],
```

Comment:
We shouldn't need the protocol here if generated enums aren't used in wlroots' public header.

Reply:
Ah, good point.